spec/models/team_spec.rb

```diff
@@ -422,42 +422,46 @@
   end
 
   describe '#display_name' do
-    context 'with a project name' do
-      before { subject.project_name = 'Sinatra' }
+    subject(:display_name) { team.display_name }
 
-      it 'returns "Team Sinatra" if no name given' do
-        expect(subject.display_name).to eql 'Team Sinatra'
-      end
+    let(:team)    { build(:team, :in_current_season, name: 'Blue') }
+    let(:project) { build(:project, name: 'Sinatra') }
 
-      it 'returns "Team Blue" if name given' do
-        subject.name = 'Blue'
-        expect(subject.display_name).to eql 'Team Blue (Sinatra)'
-      end
+    it 'returns "Team" and the name' do
```

Comment:
I think it would be even more clear if you put this spec together with the Team Team name spec (on line 434) in one context together. 
Also, I like it when specs start with the 'complete happy path', and the special cases unfolding afterwards. In this case: the display_name with team name, current season and project name. I think it would be an easier read here.

Reply:
Ok. Do you mean putting together as in merging both into one example, or just positioning them next to each other?

Reply:
I meant positioning them next to each other, within one context, like  `context "with a team name"`

Reply:
Ok. The `team` model has a presence validation on the `name` attribute, so a context like _"without a name"_ seems obsolete anyways. So I'll just remove that one.

Reply:
🤔  I think this is about two tests about how a team name is being handled for display: the handling will be different for  "Team Cheesy" or "Cheesy".  
That's two valid inputs for the display method, so I thought it makes sense to have them both AND put them in one context together. No?

Reply:
I may still be missing the point, sorry 😹 🙈 

I have both tests in place, but I don't understand why I should put them in the same context. They have a different setup (they need two different teams) - so there's no point to put them in the same context. Or do you mean something different?